modules/microshift-auto-recovery-example-ostree-systems.adoc

```diff
@@ -4,49 +4,59 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-automation-example-ostree-systems_{context}"]
-= Automating the integration process with systemd for OSTree systems
+= Using auto recovery with {op-system-ostree}
+
+As a use case, consider the following example situation in which you want to automate the `auto-recovery` process for {op-system-ostree-first} systems that use systemd in the blueprint file.
 
 [IMPORTANT]
 ====
-You must include the entire `auto-recovery` process for OSTree systems that use `systemd` in the blueprint file.
+You must include the entire `auto-recovery` process for {op-system-ostree} systems that use `systemd` in the blueprint file.
 ====
 
-As a use case, consider the following example situation in which you want to automate the `auto-recovery` process for OSTree systems that use systemd.
+.Prerequisites
+
+* You installed podman.
+* You installed the command-line `composer-cli` tool.
 
 .Procedure
 
+. Optional. Because the `composer-cli` can only create files in the `/etc` directory, package your files into an RPM that you include the blueprint.
```

Comment:
```suggestion
. Optional: Because the `composer-cli` tool can only create files in the `/etc` directory, package your files into an RPM that you include the blueprint.
```